storage.go

```diff
@@ -101,8 +101,18 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
 	rp := s.getAbsPath(path)
 
-	// `AppendObject` with position 0 will overwrite the existing object, so there's no need to check the object exists or not.
+	// If the object not exists, we should set the offset to 0.
 	var offset int64 = 0
+
+	// If the object exists, we should set the offset to the next append position.
+	headInput := &service.HeadObjectInput{}
+	headOutput, err := s.bucket.HeadObjectWithContext(ctx, rp, headInput)
```

Comment:
As described in GSP-134: https://github.com/beyondstorage/go-storage/blob/master/docs/rfcs/134-write-behavior-consistency.md#services-implementations

We need to delete this file instead of update append position.

Reply:
Oh, you are right, I should delete this object.